dist/geolib.js

```diff
@@ -937,11 +937,19 @@
         *
         * @param        object      reference coordinate e.g. {latitude: 51.5023, longitude: 7.3815}
         * @param        mixed       array or object with coords [{latitude: 51.5143, longitude: 7.4138}, {latitude: 123, longitude: 123}, ...]
+        * @param        boolean     optional flag to include reference coordinate in returned array
         * @return       array       ordered array
         */
-        orderByDistance: function(latlng, coords) {
+        orderByDistance: function(latlng, coords, includeSource) {
 
             var coordsArray = [];
+            
+            if (includeSource) {
+                var sourceObj = latlng;
```

Comment:
latlng is not necessarily an object but can also be an array of [lng, lat]. So i think sourceObj.distance would not work in such a case. Maybe we should do something like `var sourceObj = Object.create(latlng);` instead?! I will have to test this.